frontend/admin/src/js/components/pool/CreatePool.tsx

```diff
@@ -129,6 +129,16 @@ export const CreatePoolForm: React.FunctionComponent<CreatePoolFormProps> = ({
     }),
   );
 
+  const operationalRequirementsSubset = [
```

Comment:
Should we just say `operationalRequirementsV2` to hint that this is the new hotness? I'd def advocate we air toward using V1, V2, V3 suffixes as we evolve things in the app, rather than arbitrary modifier words

Reply:
```suggestion
  const operationalRequirementsV2 = [
```